src/main/scala/chisel3/tester/internal/TestEnvInterface.scala

```diff
@@ -13,6 +18,24 @@ trait TestEnvInterface {
 
   def topFileName: Option[String]
 
+  /**
+    * Will add a TargetDirAnnotation with defaultDir with "test_run_dir" path prefix to the annotations
+    * if there is not a TargetDirAnnotation already present
+    *
+    * @param defaultDir     a default directory
+    * @param annotationSeq  annotations to add it to, unless one is already there
+    * @return
+    */
+  def addDefaultTargetDir(defaultDir: String, annotationSeq: AnnotationSeq): AnnotationSeq = {
```

Comment:
I'd prefer this to fail noisily. It should be required the annotation isn't in the AnnotationSeq already. This seems to be asking for nonintuitive behavior.

Reply:
I disagree, for some tests the user may want to specify a specific working directory. The way to do that (at least for now) is to add their own `TargetDirAnnotation`